### PR TITLE
Remove illogical wording in docs

### PR DESCRIPTION
**Pull request by Andrew-Chen-Wang**
I'm not sure if you meant to say reload-includes, and I didn't check the source code, but this just seemed confusing since it says to override the argument you are specifying.

### CODE REVIEW

**Comment by Kludex** [comment]
Hey @Andrew-Chen-Wang , when I read it the first time I also felt confused. But it's actually right, and it makes sense once you read the documentation.

You can check with more details here: https://www.uvicorn.org/settings/#development

In case you have still have a better suggestion on the wording, feel free to open a PR. :)

**Comment by Andrew-Chen-Wang** [comment]
@Kludex It's definitely the wording. For instance, for `--reload-excludes`:

> These can be overwritten by explicitly including them.

I think it would make sense if it simply says: `These defaults can be overwritten by including them in --reload-include`.

So the changes are "These defaults" (so I know "these" points to the defaults and not the flag arguments itself if I specify the reload-exclude flag) and "--reload-include" instead of `reload-excludes` or something related. I'd much rather see the CLI flag explicitly.

**Comment by Kludex** [comment]
I see, you're right.

Thanks for insisting, and sorry for my bad reading skills. :sweat_smile: 

The changes on the file you changed are made using a script, that should run after a modification on the `main.py` is done.

**Comment by Andrew-Chen-Wang** [comment]
Thanks for understanding; my speaking skills are worse :P I've changed main.py; for the development docs, do I change this file directly: https://github.com/encode/uvicorn/blob/master/docs/settings.md

**Comment by Kludex** [comment]
Nop, you should run `./scripts/lint`. It will format the documentation for you!

**Comment by Andrew-Chen-Wang** [comment]
Ah sorry, I meant I changed main.py for the docs/index.md, but should I change the docs/settings.md file directly for the other wording?

**Comment by Kludex** [comment]
Oh! I forgot about this file. Go ahead, please.

**Comment by Andrew-Chen-Wang** [comment]
👍 thanks @Kludex

**Comment by Kludex** [comment]
No! Thank you! :tada:

File: docs/settings.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ you should put `uvicorn.run` into `if __name__ == '__main__'` clause in the main
 
 * `--reload` - Enable auto-reload.
 * `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default the whole current directory will be watched. If you are running programmatically use `reload_dirs=[]` and pass a list of strings.
-* `--reload-include <glob-pattern>` - Specify a glob pattern to match files or directories which will be watched. May be used multiple times. By default the following patterns are included: `*.py`. These can be overwritten by explicitly excluding them.
-* `--reload-exclude <glob-pattern>` - Specify a glob pattern to match files or directories which will excluded from watching. May be used multiple times. By default the following patterns are excluded: `.*, .py[cod], .sw.*, ~*`. These can be overwritten by explicitly including them.
+* `--reload-include <glob-pattern>` - Specify a glob pattern to match files or directories which will be watched. May be used multiple times. By default the following patterns are included: `*.py`. These defaults can be overwritten by including them in --reload-exclude.
```
Comment (Kludex) [inline docs/settings.md]:
```suggestion
* `--reload-include <glob-pattern>` - Specify a glob pattern to match files or directories which will be watched. May be used multiple times. By default, the following patterns are included: `*.py`. These defaults can be overwritten by including them in `--reload-exclude`.
```

File: docs/settings.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ you should put `uvicorn.run` into `if __name__ == '__main__'` clause in the main
...
+* `--reload-exclude <glob-pattern>` - Specify a glob pattern to match files or directories which will excluded from watching. May be used multiple times. By default the following patterns are excluded: `.*, .py[cod], .sw.*, ~*`. These defaults can be overwritten by including them in --reload-include.
```
Comment (Kludex) [inline docs/settings.md]:
```suggestion
* `--reload-exclude <glob-pattern>` - Specify a glob pattern to match files or directories which will excluded from watching. May be used multiple times. By default, the following patterns are excluded: `.*, .py[cod], .sw.*, ~*`. These defaults can be overwritten by including them in `--reload-include`.
```

File: uvicorn/main.py [hunk 3]
```diff
@@ -85,14 +85,15 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
...
-    "by default, which can be overridden in reload-excludes.",
+    "by default; these defaults can be overridden in --reload-exclude.",
 )
 @click.option(
     "--reload-exclude",
     "reload_excludes",
     multiple=True,
     help="Set glob patterns to exclude while watching for files. Includes "
-    "'.*, .py[cod], .sw.*, ~*' by default, which can be overridden in reload-excludes.",
+    "'.*, .py[cod], .sw.*, ~*' by default; the defaults can be overridden "
```
Comment (Kludex) [inline uvicorn/main.py]:
```suggestion
    "'.*, .py[cod], .sw.*, ~*' by default; these defaults can be overridden "
```

**Review by Kludex — CHANGES_REQUESTED** [review]
I've just noticed that the documentation uses quotes to reference to other commands, would you mind applying those? :pray:

**Review by Kludex — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @Andrew-Chen-Wang ! :tada:

**Pull request merged**